From 6be690b7c044878d89e9e736bbc8e816b416f02c Mon Sep 17 00:00:00 2001
From: Robert Nelson <robertcnelson@gmail.com>
Date: Thu, 17 Sep 2015 15:44:20 -0500
Subject: [PATCH 22/97] apply backport patch
 collateral-evolutions/network/0003-cfg80211-wext-padding/include_net_cfg80211.patch

---
 backports/include/net/cfg80211.h | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/backports/include/net/cfg80211.h b/backports/include/net/cfg80211.h
index f8d6813..fd2e593 100644
--- a/backports/include/net/cfg80211.h
+++ b/backports/include/net/cfg80211.h
@@ -3120,6 +3120,9 @@ struct wiphy_vendor_command {
 struct wiphy {
 	/* assign these fields before you register the wiphy */
 
+#define WIPHY_COMPAT_PAD_SIZE	2048
+	u8 padding[WIPHY_COMPAT_PAD_SIZE];
+
 	/* permanent MAC address(es) */
 	u8 perm_addr[ETH_ALEN];
 	u8 addr_mask[ETH_ALEN];
-- 
2.5.1

